cmd/wrapper/cmd/mainnet.go

```diff
@@ -527,3 +590,117 @@ func parseClaimRecords(genParams *GenesisStates) ([]claimtypes.ClaimRecord, []ba
 
 	return records, balances, totalInitialGenesisCoin
 }
+
+func ParseVestingAccounts(filePath string) (sdk.Int, map[string]*authvesting.PeriodicVestingAccount, []*authvesting.PeriodicVestingAccount) {
+	vestingAccs := []*authvesting.PeriodicVestingAccount{}
+	vestingAccMap := make(map[string]*authvesting.PeriodicVestingAccount)
+	results, err := readCSVFile(filePath)
+	if err != nil {
+		panic(fmt.Sprintf("failed to read csv file %s", filePath))
+	}
+
+	totalVestingAmt := sdk.ZeroInt()
+
+	for i, r := range results {
+		if i == 0 {
+			continue
+		}
+
+		recipientAddr := r[0]
+		vestingAmt, _ := sdk.NewIntFromString(r[1])
+
+		// Convert bech32 address prefix
+		_, converted, err := bech32.DecodeAndConvert(recipientAddr)
+		if err != nil {
+			panic(err)
+		}
+
+		targetPrefix := sdk.GetConfig().GetBech32AccountAddrPrefix()
+		recipientAddr, err = bech32.ConvertAndEncode(targetPrefix, converted)
+		if err != nil {
+			panic(err)
+		}
+		recipientAcc, err := sdk.AccAddressFromBech32(recipientAddr)
+		if err != nil {
+			panic(err)
+		}
+
+		// Skip the zero amount
+		if vestingAmt.IsZero() {
+			continue
+		}
+
+		baseAcc := authtypes.NewBaseAccount(recipientAcc, nil, 0, 0)
+		periodVestingAcc := authvesting.NewPeriodicVestingAccount(baseAcc, sdk.NewCoins(sdk.NewCoin(BondDenom, vestingAmt)), GenesisTimeUnix, CalcVestingPeriod(vestingAmt))
+		vestingAccMap[periodVestingAcc.Address] = periodVestingAcc
```

Comment:
Currently, there is no duplicate address for testing temporary vesting.csv with airdrop recipient, but the logic was created because it could be added later